components/EliminateForm.jsx

```diff
@@ -0,0 +1,43 @@
+import StyledForm from "@/components/Styles/StyledForm";
+import StyledDiv from "@/components/Styles/StyledDiv";
+import PositionedButton from "@/components/Styles/StyledButton";
+import styled from "styled-components";
+
+export default function EliminateForm({ onEliminate, onMode, onSubmit }) {
+  return (
+    <div style={{ position: "fixed", bottom: "0", width: "100%" }}>
```

Comment:
why inline styling here, when we use styled-components?

Reply:
You are right. We totally forgot about that. Thanks for the hint!